crypto/err/err.c

```diff
@@ -92,17 +93,16 @@ static ERR_STRING_DATA ERR_str_reasons[] = {
     {ERR_R_DSA_LIB, "DSA lib"},
     {ERR_R_X509_LIB, "X509 lib"},
     {ERR_R_ASN1_LIB, "ASN1 lib"},
+    {ERR_R_CRYPTO_LIB, "crypto lib"},
     {ERR_R_EC_LIB, "EC lib"},
     {ERR_R_BIO_LIB, "BIO lib"},
     {ERR_R_PKCS7_LIB, "PKCS7 lib"},
     {ERR_R_X509V3_LIB, "X509V3 lib"},
-    {ERR_R_ENGINE_LIB, "ENGINE lib"},
+    {ERR_R_ENGINE_LIB, "engine lib"},
```

Comment:
?

Reply:
Same reason as above

Reply:
The `ENGINE` library is the set of functions with the `ENGINE` (not `engine` prefix).